zephyr/CMakeLists.txt

```diff
@@ -734,23 +694,18 @@ zephyr_library_sources_ifdef(CONFIG_COMP_CHAIN_DMA
 	${SOF_AUDIO_PATH}/chain_dma.c
 )
 
-if(CONFIG_IPC_MAJOR_3)
-	zephyr_library_sources_ifdef(CONFIG_COMP_SRC
-		${SOF_AUDIO_PATH}/src/src_hifi2ep.c
-		${SOF_AUDIO_PATH}/src/src_generic.c
-		${SOF_AUDIO_PATH}/src/src_hifi3.c
-		${SOF_AUDIO_PATH}/src/src_hifi4.c
-		${SOF_AUDIO_PATH}/src/src.c
-		${SOF_AUDIO_PATH}/src/src_ipc3.c
-	)
-elseif(CONFIG_IPC_MAJOR_4)
-	zephyr_library_sources_ifdef(CONFIG_COMP_SRC
+if(CONFIG_COMP_SRC STREQUAL "m")
+	add_subdirectory(${SOF_AUDIO_PATH}/src/llext
+			 ${PROJECT_BINARY_DIR}/src_llext)
+	add_dependencies(app src)
+elseif(CONFIG_COMP_SRC)
+	zephyr_library_sources(
```

Comment:
I think this is adding an unrelated change related to llext...

Reply:
@kv2019i oh, just a tiny small one, wouldn't hurt! ;-) No, thanks for catching, indeed!